main.js

```diff
@@ -22,6 +22,25 @@ document.addEventListener('DOMContentLoaded', function(e) {
     };
 
     const successFunction = (response) => {
+
+        // decode TXT buffer
+        for (const a of response['answers']) {
+            if (a.type === 'TXT') {
+                a.data = a.data.toString();
+            }
+        }
+        // replace padding buffer with length
+        for (const a of response['additionals']) {
```

Comment:
We could theoretically call `toString()` on the padding as well. Just a thought.

Reply:
I thought about that, but usually the values are just 0 for padding. So `toString()` would decode to some weird control characters.